.env.example

```
@@ -0,0 +1,56 @@
+DB_NAME=onecurrency
+
+# Next.js Configuration
+NEXT_PUBLIC_API_URL=
+
+# Blockchain Configuration
+NEXT_PUBLIC_CHAIN_ID=31337
+NEXT_PUBLIC_WALLET_CONNECT_ID=
+
+# Neon Configuration
+DIRECT_DATABASE_URL_='postgresql://neondb_owner:npg_7KZSGg5txrqH@ep-cold-tree-a1u1bzbe.ap-southeast-1.aws.neon.tech/neondb?sslmode=require&channel_binding=require'
+DATABASE_URL='postgresql://neondb_owner:npg_7KZSGg5txrqH@ep-cold-tree-a1u1bzbe-pooler.ap-southeast-1.aws.neon.tech/neondb?sslmode=require&channel_binding=require'
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The DATABASE_URL key should go before the DIRECT_DATABASE_URL_ key

(UnorderedKey)